src/PrestaShopBundle/Resources/views/Admin/Module/Includes/card_grid.html.twig

```diff
@@ -0,0 +1,81 @@
+{% set isModuleActive = module.database.active|default('0') %}
+{% set productType = module.attributes.serviceUrl is defined and module.attributes.serviceUrl|length > 0 ? 'service' : 'module' %}
+
+<div
+  class="module-item module-item-grid col-md-12 col-lg-6 col-xl-4 {% if origin == 'manage' and isModuleActive == '0' %}module-item-grid-isNotActive{% endif %}"
+  data-id="{{ module.attributes.id }}"
+  data-name="{{ module.attributes.displayName }}"
+  data-scoring="{{ module.attributes.avgRate }}"
+  data-logo="{{ module.attributes.img }}"
+  data-author="{{ module.attributes.author }}"
+  data-version="{{ module.attributes.version }}"
+  data-description="{{ module.attributes.description }}"
+  data-tech-name="{{ module.attributes.name }}"
+  data-child-categories="{{ module.attributes.categoryName }}"
+  data-categories="{{ module.attributes.categoryParent }}"
+  data-type="{{ productType }}"
+  data-price="{{ module.attributes.price.raw }}"
+  data-active="{{ isModuleActive }}"
+>
+  <div class="module-item-wrapper-grid">
+    <div class="module-item-heading-grid">
+      <div class="module-logo-thumb-grid">
+        <img src="{{ module.attributes.img }}" alt="{{ module.attributes.displayName }}"/>
+      </div>
+      <h3
+        class="text-ellipsis module-name-grid"
+        data-toggle="tooltip"
+        data-placement="top"
+        title="{{ module.attributes.displayName|raw }}"
+      >
+        {% if module.attributes.displayName %}
+          {{ module.attributes.displayName|raw }}
+        {% else %}
+          {{ module.attributes.name }}
+        {% endif %}
+      </h3>
+      <div class="text-ellipsis xsmall module-version-author-grid">
+        {% block addon_version %}
+          {% if productType == "service" %}
+            {{ 'Service by '|trans({}, 'Admin.Modules.Feature') }}
+            <b>{{ module.attributes.author }}</b>
+          {% else %}
+            {{ 'v%version% - by '|trans({ '%version%' : module.attributes.version }, 'Admin.Modules.Feature') }}
+            <b>{{ module.attributes.author }}</b>
+          {% endif %}
+        {% endblock %}
+      </div>
+    </div>
+    <div class="module-quick-description-grid small no-padding">
+      {% block addon_description %}
+        <div class="module-quick-description-text">
+          {{ module.attributes.description|raw }}
+          {% if module.attributes.description|length > 0 and module.attributes.description|length < module.attributes.fullDescription|length %}
+            ...
```

Comment:
For your information:

On the Addons API, we have two properties `description` and `fullDescription`. They have almost the same content, but `description` has a limited number of characters. Because the three dots do not exist if the fullDescription is longer, we add them in the view.